spec

```diff
@@ -1720,8 +1720,11 @@ beginseg
     name "ovl_En_Fu"
     compress
     include "build/src/overlays/actors/ovl_En_Fu/z_en_fu.o"
-    include "build/data/ovl_En_Fu/ovl_En_Fu.data.o"
-    include "build/data/ovl_En_Fu/ovl_En_Fu.reloc.o"
+    #ifdef NON_MATCHING
+        include "build/src/overlays/actors/ovl_En_Fu/ovl_En_Fu_reloc.o"
+    #else
+        include "build/data/ovl_En_Fu/ovl_En_Fu.reloc.o"
+    #endif
```

Comment:
Are there non-matchings then? Could you specify in the title or the description how many non-matchings there are?